targets/unity-v2/source/Shared/External/aria/Schema3_types.cs

```diff
@@ -684,19 +697,24 @@ internal void Reset()
             extApp = null;
             extUtc = null;
             extNet = null;
-            extDevice = null;
```

Comment:
> extDevice = null; [](start = 11, length = 18)

I understand the property is duplicated but does this not change the event schema ? Have we validated that we can send this data and see it in ARIA protal ?